shell/i18n/en.i18n.json

```diff
@@ -344,44 +344,47 @@
           "hasApps": "Select an app and create your first grain.",
           "hasNoApp": "Install an app to create your first grain."
         },
-        "transfers": {
```

Comment:
My question was just regarding this change. Whether or not the right fix was to remove this "grouping" for transfers, or to add it to the places where these strings are invoked.